projects/vdk-control-cli/.gitlab-ci.yml

```diff
@@ -41,6 +41,10 @@ vdk-control-cli-build-with-py310:
   image: "python:3.10"
   extends: .vdk-control-cli-build
 
+vdk-control-cli-build-with-py311:
+    image: "python:3.11"
+    extends: .vdk-control-cli-build
+
 vdk-control-cli-release-acceptance-test:
```

Comment:
some plugin directories are missing .gitlab-ci.yaml file modification.
Is the change actually modifying two files in each plugin dir, `setup.cfg` and `.gitlab-ci.yml`? If yes, you could just say so in the PR description

Reply:
Updated